airflow/www/templates/airflow/dags.html

```diff
@@ -49,8 +49,17 @@
                 <td style="padding:0px; width:180px; height:10px;">
                     <svg height="10" width="10" id='{{ dag.dag_id }}' style="display: block;"></svg>
                 </td>
-                <td class="text-center" style="width:160px;">
+                <td class="text-center" style="width:180px;">
                 {% if dag %}
+                  {% if dag_id in paused_dags %}
+                    <a href="{{ url_for("airflow.paused", dag_id=dag.dag_id) }}" title="Paused">
+                        <span class="glyphicon glyphicon-pause" aria-hidden="true"></span>
```

Comment:
`title="Click this to pause this DAG"`

These can be confusing as the pause sign can be interpreted as an action or as a state, not sure how to make that 100% clear